src/SearchResults.js

```diff
@@ -0,0 +1,56 @@
+import moment from "moment"
+import React, { useState } from "react"
+
+
+const SearchResults = (props) => {
+    let allData = props.results
+    return (
+        <table class="table table-striped">
+
+            <thead>
+
+                <tr className="title-table">
+                    <th scope="col">ID</th>
+                    <th scope="col">Title</th>
+                    <th scope="col">First Name</th>
+                    <th scope="col">Surname</th>
+                    <th scope="col">Email</th>
+                    <th scope="col">Room ID</th>
+                    <th scope="col">Check In Date</th>
+                    <th scope="col">Check Out Date</th>
+                    <th scope="col">Nights</th>
+                </tr>
+            </thead>
+            <tbody>
+                {
+
+                    allData.map(client => {
+                        let a = moment(client.checkOutDate)
+                        let b = moment(client.checkInDate)
```

Comment:
nit: it's better to have a more descriptive variable names.

let checkOutDate = moment(client.checkOutDate)
let checkInDate = moment(client.checkInDate)

Reply:
thanks will do)